extensions/http-curl/tests/unit/InvokeHTTPTests.cpp

```diff
@@ -306,3 +291,97 @@ TEST_CASE("HTTPTestsPostNoResourceClaim", "[httptest1]") {
   REQUIRE(true == LogTestController::getInstance().contains("Exiting because method is POST"));
   LogTestController::getInstance().reset();
 }
+
+TEST_CASE("HTTPTestsPenalizeNoRetry", "[httptest1]") {
+  TestController testController;
+  using processors::InvokeHTTP;
+
+  std::shared_ptr<core::ContentRepository>
+  content_repo = std::make_shared<core::repository::VolatileContentRepository>();
```

Comment:
This is a single statement. It should be on one line in this case, but in general it can be separated with a new line + continuation indentation (+4 spaces or aligned) if it's too long.

Reply:
Okay I will change it. For some reason, it keeps happening on its own. Is there any way to make sure it doesn't happen anymore?

Reply:
Maybe your editor/IDE is enforcing some line length requirements? In CLion, it's under Settings/Editor/Code Style/{Hard Wrap At,Wrap on typing}.